### PR TITLE
Search Tv shows and All Tv shows functionality

### LINKED ISSUE #16: Implement navbar search functionality

### LINKED ISSUE #17: Add navigation links functionality to the navbar

### PR DESCRIPTION
**Pull request by tennjugu**
This PR closes issues #16 and #17

### CODE REVIEW

File: .hintrc
Comment (Bazzy09) [inline .hintrc:1]: good stuff

**Review by Bazzy09 — APPROVED** [review]

File: dist/main.js
Comment (Bazzy09) [inline dist/main.js:1]: this code is well structured, and clearly written.

**Review by Bazzy09 — APPROVED** [review]

**Review by Bazzy09 — APPROVED** [review]

**Review by Mogle7Arkad — APPROVED** [review]
A  nice and clean code.  good naming convention too.

**Comment by tennjugu** [comment]
Thanks for the positive feedback.

**Pull request merged**